database/src/main/resources/db/migration/V76__begrunnelse_i_beregning.sql

```diff
@@ -0,0 +1,2 @@
+update behandling set beregning = jsonb_set(beregning, '{begrunnelse}', '"migrert"') where beregning #>> '{begrunnelse}' is null;
```

Comment:
Håndterer vi denne "migrert"-verdien i frontend (og erstatter den med tom streng eller noe sånt), eller viser vi bare "migrert" i begrunnelses-feltet på en åpen behandling? Er ikke sikkert det sistnevnte er noe problem, men kan jo kjøre det via Tor Erik evt.

Reply:
Ja, migrert blir vist i begrunnelses feltet. den var egentlig bare for å teste at den faktisk funket. Kunne forsåvidt legge til en tom streng istedenfor. Da vil ikke noen behandlinger egentlig endre seg

Reply:
Tidligere da vi la dropdown og tekstfelt (årsak og begrunnelse) når man starter en ny revurdering, la jeg inn disse som defaults:
```
alter table revurdering
    add column if not exists
    årsak text default 'MIGRERT';

alter table revurdering
    add column if not exists
    begrunnelse text default 'MIGRERT';
```

Reply:
Tror vi glemte spørre fag dog :grimacing: